src/sentry/static/sentry/app/components/forms/selectControl.jsx

```diff
@@ -113,6 +113,10 @@ const StyledSelect = styled(React.forwardRef(forwardRef))`
     }
   }
 
+  &.Select--single.is-disabled .Select-control .Select-value .Select-value-label {
```

Comment:
Does this apply to multi select as well?

Reply:
It doesn't. With these changes a disabled multi-select looks like:

![screen shot 2018-12-04 at 4 44 27 pm](https://user-images.githubusercontent.com/24086/49474932-ed8edd80-f7e3-11e8-9fff-41cd392993b8.png)